src/lib/core/grafana_config_generator.py

```diff
@@ -1,73 +1,191 @@
 import os
 import json
+import logging
 
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
 
-def generate_dashboard_model(
-    service, timestream_database_name, timestream_table_name
+
+def replace_values_in_json(json_data: json, replacements: dict) -> dict:
+    """
+    Replaces values in a JSON object based on a dictionary of replacements.
+
+    Args:
+        json_data (dict): The JSON object.
+        replacements (dict): Dictionary with keys as the values to be replaced and
+                            corresponding values as the replacements.
+
+    Returns:
+        dict: Updated JSON object.
+    """
+
+    def replace_recursive(obj):
+        if isinstance(obj, dict):
+            for key, value in obj.items():
+                obj[key] = replace_recursive(value)
+        elif isinstance(obj, list):
+            for i, item in enumerate(obj):
+                obj[i] = replace_recursive(item)
+        elif isinstance(obj, str) and obj in replacements:
+            obj = replacements[obj]
+        return obj
+
+    return replace_recursive(json_data)
```

Comment:
We have the same replacements use-case and implementation in settings. It makes sense to have it only once. 
Can you please move this method in lib.core.json_utils, so that I can also use it in settings afterwards not to duplicate the code?
Thanks in advance!

Reply:
thank you for your comment, sure :)

Reply:
updated accordingly